app/main.py

```diff
@@ -16,4 +16,58 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+"""
+# parse the names of first n variables in a file
+def get_n_variable_names(file, n):
+    try:
+        with open(file, "r") as f:
+            variables = []
+            while len(variables) != n:
+                line = f.readline().split(" = ")[0]
```

Comment:
The current implementation assumes that each line contains exactly one variable assignment in the format 'variable = value'. If there are lines with different formats or comments, this logic might fail. Consider adding additional checks or parsing logic to handle such cases.